sql-plugins/src/main/java/io/cdap/pipeline/sql/plugins/bigquery/BigQueryUtil.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.pipeline.sql.plugins.bigquery;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Table;
+import com.google.cloud.bigquery.TableId;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import javax.annotation.Nullable;
+
+/**
+ * BigQuery utility functions class.
+ */
+public class BigQueryUtil {
+  /**
+   * Loads Google credentials from a service account file path.
+   *
+   * @param serviceAccountPath The path to the service account identity file
+   * @return The credentials object
+   */
+  public static GoogleCredentials loadServiceAccountCredentials(String serviceAccountPath) {
+    GoogleCredentials credentials;
+    try {
+      File credentialsPath = new File(serviceAccountPath);
+      FileInputStream serviceAccountStream = new FileInputStream(credentialsPath);
+      credentials = ServiceAccountCredentials.fromStream(serviceAccountStream);
+    } catch (IOException e) {
+      throw new IllegalArgumentException("Invalid credentials file");
+    }
+    return credentials;
+  }
+
+  /**
+   * Initializes and returns a BigQuery service object from a passed in Google credential object.
+   *
+   * @param credentials The Google credential object to authenticate as
+   * @return The BigQuery service object
+   */
+  public static BigQuery initializeBigQuery(@Nullable GoogleCredentials credentials) {
+    BigQueryOptions.Builder builder = BigQueryOptions.newBuilder();
+    if (credentials != null) {
+      builder.setCredentials(credentials);
+    }
+    return builder.build().getService();
+  }
+
+  /**
+   * Get BigQuery table.
+   *
+   * @param projectId BigQuery project ID
+   * @param datasetId BigQuery dataset ID
+   * @param tableName BigQuery table name
+   * @param serviceAccountPath service account file path
+   * @return BigQuery table
+   */
+  @Nullable
+  public static Table getBigQueryTable(String projectId, String datasetId, String tableName,
```

Comment:
I don't think this is a good method to have, as it encourages the creation of multiple BigQuery clients (within this method) instead of creating it once and using it repeatedly

Reply:
Fixed by removing the BigQueryUtil